packages/common/container-definitions/src/deltas.ts

```diff
@@ -123,7 +123,7 @@ export interface IDeltaManagerEvents extends IEvent {
 	(event: "allSentOpsAckd", listener: () => void);
 
 	/**
-	 * @deprecated No replacement API recommended.
+	 * Emitted from client to server, purpose is tracking latency
```

Comment:
I do not think at this layer it matters who emits events (it's implementation details).
What matters here is that this event is raised periodically with latest information on network roundtrip latency - it would be great to capture it in comments.